File: python/with_java/programmers/level 2/행렬의 곱셈.py
def solution(arr1, arr2):
    row_len = len(arr1)
    col_len = len(arr2[0])
    mix = len(arr2)
    answer = []
    for i in range(row_len):
        row = []
        for j in range(col_len):
            result = 0
            for k in range(mix):
                result += arr1[i][k] * arr2[k][j]
            row.append(result)
        answer.append(row)
    return answer

File: java/src/main/java/ps_traning/programmers/level_2/행렬의_곱셈.java
package ps_traning.programmers.level_2;

public class 행렬의_곱셈 {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int rowLen = arr1.length;
        int colLen = arr2[0].length;
        int mix = arr2.length;
        int[][] answer = new int[rowLen][colLen];
        for (int i = 0; i < rowLen; i++) {
            for (int j = 0; j < colLen; j++) {
                int result = 0;
                for (int k = 0; k < mix; k++) {
                    result += arr1[i][k] * arr2[k][j];
                }
                answer[i][j] = result;
            }
        }
        return answer;
    }
}
